powrap/powrap.py

```diff
@@ -27,7 +28,10 @@ def check_style(po_files: Iterable[str], no_wrap=False, quiet=False, diff=False)
         except OSError as open_error:
             tqdm.write(f"Error opening '{po_path}': {open_error}")
             continue
-        with NamedTemporaryFile("w+") as tmpfile:
+
+        delete = os.name == "posix" and sys.platform != "cygwin"
+
+        with NamedTemporaryFile("w+", delete=delete, encoding="utf-8") as tmpfile:
```

Comment:
Workaround for https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile:

> Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows). If delete is true (the default), the file is deleted as soon as it is closed.

We simply don't delete the file after it is created, making it possible to write into it while it is still open. (For non-Posix systems)